Libraries/Utilities/MessageQueue.js

```diff
@@ -168,10 +170,11 @@ class MessageQueue {
   __nativeCall(module, method, params, onFail, onSucc) {
     if (onFail || onSucc) {
       if (__DEV__) {
-        // eventually delete old debug info
-        (this._callbackID > (1 << 5)) &&
-          (this._debugInfo[this._callbackID >> 5] = null);
-        this._debugInfo[this._callbackID >> 1] = [module, method];
+        let debugId = this._callbackID >> 1;
```

Comment:
Maybe rename this to `callId`, it took me a few moments to realize the bitshift is for the fact that the native calls have two callbacks.